client/common/services/FileUploadService.js

```diff
@@ -8,12 +8,14 @@ angular.module('dashboard.services.FileUpload', [
 .service('FileUploadService', function($cookies, $http, $q, $upload, Config, Utils) {
 
   var self = this;
-  
+  var uploadACL = Config.serverParams.unsafeFileUpload ? 'public-read' : 'private';
```

Comment:
I think we should call the parameter something like: serverParams.defaultACL

There are use cases where we want the files to be public. If for example, we have a simple CMS to manage video content for the client to download. And there are use cases where we want the files to be private.

If defaultACL is not set, we should leave it as 'public-read' for backwards compatibility.